repl.js

```diff
@@ -132,7 +141,7 @@ function usage(){
 function start(){
   console.log('… ready!');
   var cmd = repl.start({
-    prompt: str + ' › ',
+    prompt: '\033[96m' + str + ' › \033[39m',
```

Comment:
Maybe we could test if the version of node running supports the colored prompt or not? I believe that v0.12.2 was the first version to include the fix.